src/opera/test/data/test_disp_s1_config.yaml

```diff
@@ -25,6 +25,7 @@ RunConfig:
                         - disp_s1_pge_test/input_dir/jplg0410.18i.Z
                     troposphere_files:
                         - disp_s1_pge_test/input_dir/GMAO_tropo_20180210T000000_ztd.nc
+                        - disp_s1_pge_test/input_dir/GMAO_tropo_20180211T000000_ztd.nc
```

Comment:
For consistency, these should be set to the same set of filenames used in the SAS/dynamic_ancillary_file_group/troposphere_files section below